README.md

```diff
@@ -374,13 +376,43 @@ let app =
         ]
 ```
 
+### subRoute
+
+`subRoute` checks the current path begins with the `path` string, all `route*` and `subRoute*` handlers within the `handler` parameter will now be rooted to, and relative with respect to this path.
+
+#### Example:
+
+```fsharp
+let app = 
+    subRoute "/api" >>=
+        choose [
+            route "/v1/foo" >>= text "Foo"
+            route "/v1/bar" >>= text "Bar"
+        ]
+```
+
```

Comment:
Maybe also add this below to show nested sub routes:

You can also nest multiple `subRoute` handlers:

```fsharp
let app = 
    subRoute "/api" >>=
        choose [
            subRoute "/v1" >>=
                choose [
                    route "/foo" >>= text "Foo 1"
                    route "/bar" >>= text "Bar 1" ]
            subRoute "/v2" >>=
                choose [
                    route "/foo" >>= text "Foo 2"
                    route "/bar" >>= text "Bar 2" ] ]
 ```